modules/navier_stokes/test/tests/finite_volume/cns/pressure_outlet/subsonic_nozzle_fixed_inflow_hllc.i

```diff
@@ -0,0 +1,315 @@
+inlet_vel = 1.2
+rho_in = 0.8719748696
+H_in = 4.0138771448e+05
+
+mass_flux = ${fparse inlet_vel * rho_in}
+enthalpy_flux = ${fparse inlet_vel * rho_in * H_in}
+
+outlet_pressure = 0.9e5
+
+[GlobalParams]
+  fp = fp
+[]
+
```

Comment:
```suggestion
```
might be simpler at this point

Reply:
It was missing from the material so I actually kept this block and flushed all the other `fp =` lines